src/vm/posix_thread.c

```diff
@@ -238,31 +238,6 @@ void os_thread_exit(void *retval)
 uint8 *os_thread_get_stack_boundary()
```

Comment:
https://github.com/deeplang-org/deeplang/pull/25/files#diff-6a93a9dcf29f0c1cb4d4d82e28f32204
Does this means we do not need this function...?